colabs/Pandas_Profiling .ipynb

```diff
@@ -0,0 +1,10911 @@
+{
+  "cells": [
```

Comment:
Wnioski wpisujmy na początku notatnika. Jak streszczenie.

Reply:
Raporty umieściłbym w `reports/` zamiast '`tutorials/raporty`. Z dwóch powodów

- To nie są tutorials
- raporty to słowo polskie a nazwy katalogów są po angielsku

Reply:
Wnioski przeniesione na początek notatnika, zmiana umiejscowienia raportów.